docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -472,6 +472,17 @@ When moving modules between collections
 
 See :ref:`Migrating content to a different collection <migrate_to_collection>` for complete details.
 
+Generally we do not object to moving content between collections, or moving content from collections included in Ansible to collections outside the Ansible package, as long as semantic versioning is not violated. More precisely, replacing content by redirects is only a minor change if the destination collection is the dependency of the source collection of the move. (See :ref:`coll_dependencies` for adding new dependencies to collections included in Ansible.)
+
+For the community "catch all" collections, we have slightly different rules. We allow to move content out of community.general and community.network to other collections outside of Ansible under the following conditions:
+
+1. The new collection is appropriately licensed and does not require a CLA to contribute.
+2. None of the contributors who contributed to the content in the last 6 months objects in a four-weeks period after the plan to deprecate the module has been announced.
```

Comment:
```suggestion
2. None of the contributors who contributed to the content in the last 6 months object in a four-week period after the plan to deprecate the module has been announced.
```

Should we specify "contributors who contributed nontrivial changes" or something like that, also?

Reply:
I think it's easier (less discussion on what constitutes a nontrivial change) to simply allow all to veto.